docker.config.5g

```diff
@@ -0,0 +1,18 @@
+%% -*- erlang -*-
+[
+  "config/sys.config",
+  {lager,
+    [
+      {log_root, "/var/log/miner"}
+    ]},
+  {blockchain,
+    [
+      {key, {tpm, [{key_path, "HS/SRK/MinerKey"}]}}
```

Comment:
Where does this path come from?

Reply:
@marvinmarnold this is for the 5G miner - it is where the TPM key will be stored.

Copied it from here:
https://github.com/NebraLtd/miner/commit/f05a22fc17592485a52f9a12ca90d49ff0b66eba#diff-07e554b101a56c616e34da7e1ccddbd9cce5c4ede459d702b90d7142482ee2d9R21
https://github.com/NebraLtd/miner/tree/jma/tpm
https://github.com/openEPC/erltss2

AFAIK `HS/SRK/` is where TPM2.0 keys are stored by default using https://github.com/tpm2-software/tpm2-tools and https://github.com/tpm2-software/tpm2-tss